playground/signalr/serverless/SignalRServerless.Functions/Program.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.Azure.Functions.Worker.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+var builder = FunctionsApplication.CreateBuilder(args);
+
+builder.AddServiceDefaults();
+builder.AddAzureQueueClient("queue");
+builder.AddAzureBlobClient("blob");
+
+builder.ConfigureFunctionsWebApplication();
+
+builder.Services.AddOptions<KestrelServerOptions>()
```

Comment:
Why is this config needed?

Reply:
This config is needed to allow Functions to run synchronous IO operations (e.g reading .html file in `GetWebPage` function)